users/user-manager.js

```diff
@@ -32,33 +32,33 @@ var createUser = (json) => {
   return new response.ResponseObject(201, {"uid":"1", "url":"//www.google.com"});
 }
 
-var modifyUser = (id, json) => {
+var modifyUser = (userUid, reqData, modifyCallback) => {
   // TODO(ryanfaulkenberry100): Check if modifier is a user or an admin.
   if (/*modifying user is student*/ true) {
-    modifyUserAsSelf(id, json);
+    modifyUserAsSelf(userUid, reqData, modifyCallback);
   } else if (/*modifying user is admin*/ false) {
-    modifyUserAsAdmin(id, json);
+    modifyUserAsAdmin(userUid, reqData, modifyCallback);
   } else {
     // TODO(ryanfaulkenberry100): handle errors.
   }
 
   return new response.ResponseObject(200, {"url":"//www.google.com"});
 }
 
-var deleteUser = (id) => {
+var deleteUser = (userUid) => {
   // TODO(ryanfaulkenberry100): Delete a user and remove console.log.
-  console.log(id);
+  console.log(userUid);
 
   return new response.ResponseObject(200, {"url":"//www.google.com"});
```

Comment:
Put a TODO here to indicate that this needs to be hooked up to the `user-manager`